float64 x
float64 y
---
float64 is_goal
